### PR TITLE
Use metadata from counters as tags in AppMetrics

### PR DESCRIPTION
**Pull request by seif**
Wanted to use some metadata in graphs but they were not available, this copies the metadata from the counters to appmetrics tags.

### CODE REVIEW

**Comment by seif** [comment]
@neyromant I realise this will start adding tags to metrics that didn't have them before and could possibly existing dashboard for some people. Should I add an option so people optin to the new behaviour "SetTagsFromMetadata"  defaulting to false?

**Comment by neyromant** [comment]
> Should I add an option so people optin to the new behaviour "SetTagsFromMetadata" defaulting to false?
@seif Hi, I think yes.

File: src/Neyro.AppMetrics.Extensions.EventCountersCollector/Neyro.AppMetrics.Extensions.EventCountersCollector/_InternalVisibleTo.cs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("UnitTests")]
```
Comment (neyromant) [inline src/Neyro.AppMetrics.Extensions.EventCountersCollector/Neyro.AppMetrics.Extensions.EventCountersCollector/_InternalVisibleTo.cs]: What about put it to .csproj?
```
<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
    <_Parameter1>$(AssemblyName).Tests</_Parameter1>
</AssemblyAttribute>
```
or
```
<AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleToAttribute">
    <_Parameter1>UnitTests</_Parameter1>
</AssemblyAttribute>
```

Comment (seif) [reply src/Neyro.AppMetrics.Extensions.EventCountersCollector/Neyro.AppMetrics.Extensions.EventCountersCollector/_InternalVisibleTo.cs]: Thanks @neyromant didn't know about you can set internals in csproj. I've updated accordingly

**Comment by neyromant** [comment]
@seif Thanks a lot! I published package version 0.0.5

**Comment by seif** [comment]
Awesome, thank you @neyromant

**Pull request merged**